src/components/timeline/TrackNetwork.js

```diff
@@ -182,15 +188,23 @@ class NetworkCanvas extends PureComponent<CanvasProps> {
         if (timing.index[timingIndex] === hoveredMarkerIndex) {
           // Save the hovered path to draw it at the end, on top of everything else.
           hoveredPath = path;
+        } else if (timing.index[timingIndex] === rightClickedMarkerIndex) {
+          rightClickedPath = path;
         } else {
           ctx.stroke(path);
         }
       }
     }
 
-    if (hoveredPath) {
+    if (hoveredPath || rightClickedPath) {
       ctx.strokeStyle = HOVERED_STYLE;
-      ctx.stroke(hoveredPath);
+      if (hoveredPath) {
+        ctx.stroke(hoveredPath);
+      }
+
+      if (rightClickedPath) {
+        ctx.stroke(rightClickedPath);
+      }
```

Comment:
So we are highlighting both hovered and the right clicked, right? I wonder if we can higlight the right clicked one differently, so it could be more obvious to people. But currently we are also doing the same thing for the network chart. So we can land this, and maybe think about it if you think it's interesting?

Reply:
I thought that in this panel using the same color for hover / right click / selected (from the next PR) would be good enough. Normally the context menu should be close to the one right clicked, and the hovered one change very easily when moving the mouse, so I don't think there would be much confusion.

But that's super easy to change in the future if we feel like it. But finding the right color isn't easy :-)

Reply:
Yeah, it was just a random thought :) This looks good to me too.